utils/db_utils.go

```diff
@@ -0,0 +1,111 @@
+package oyster_utils
+
+import (
+	"fmt"
+	"github.com/gobuffalo/pop"
+	"github.com/gobuffalo/pop/columns"
+	"github.com/gobuffalo/uuid"
+	"github.com/pkg/errors"
+	"reflect"
+	"sort"
+	"strconv"
+	"strings"
+)
```

Comment:
A lot of the variables in this file could use more descriptive names, to make it easier for other developers to be able to quickly look at know what the code is doing.  No need to worry about it for this PR, could address if when you push the PR to do the batch updates every X iterations.

Reply:
Can you give me an example? I could update it.

Reply:
Yeah.  
c := columns.ColumnsForStructWithAlias(vPtr, m.TableName(), m.As)   
Could be something like:
columns := columns.ColumnsForStructWithAlias(vPtr, model.TableName(), model.As)

And something like this:  
m := pop.Model{Value: vPtr}
Could be something like this:
model := pop.Model{Value: vPtr}

Or like this:
xs = append(xs, fmt.Sprintf("'%s'", u.String()))  
It's not super obvious what it is.  I'm guessing those are strings with the column names, so maybe something like "columnName" or something.